pkg/engine/background.go

```diff
@@ -69,12 +70,30 @@ func filterRule(rclient registryclient.Client, rule kyvernov1.Rule, policyContex
 		return nil
 	}
 
+	// check if there is a corresponding policy exception
+	exception, err := matchesException(policyContext, &rule)
+	// if we found an exception
+	if err == nil && exception != nil {
+		key, err := cache.MetaNamespaceKeyFunc(exception)
+		logger := logging.WithName("exception")
+		// TODO: increase metrics
+		if err != nil {
+			logger.Error(err, "failed to compute policy exception key", "namespace", exception.GetNamespace(), "name", exception.GetName())
```

Comment:
We need to report a rule error in this case.

Reply:
Sounds good! Updated with a RuleResponse returned( assume:`Status: response.RuleStatusError`). Will that be good?